### ISSUE TITLE
What version of the browser support ?

### ISSUE THREAD

**Issue by cheft** [marko-js/marko#438]
### Open Questions
What version of the browser support ?

**Comment by Hesulan** [comment]
@cheft 

None of the built-in tags currently generate anything above ES5. You can find a detailed compatibility table for all major browsers [here](https://kangax.github.io/compat-table/es5/).

The compiler currently requires at least partial ES6 support (Node.js 4+). If you're not already compiling your templates *before* sending them to the browser, you really should.

The renderer runs in all modern browsers; if you need to support older browsers (like IE8), you will need to transpile the renderer and the compiled (".marko.js") templates with [Babel](https://babeljs.io/) before serving them to the client.

**Comment by mlrawlings** [comment]
@cheft, Marko v4 will be IE9+, so I would plan for that, but I believe we're currently IE8+ on v3.

I'm going to close this, but feel free to ask if you have more questions.

**Comment by TomHuang204** [comment]
@mlrawlings ,hi, may i ask the what version of the browser support now? we have some issues with IE by using Marko v4.

**Comment by mlrawlings** [comment]
@TomHuang204 If's IE 9 or higher, please open an issue for what you're seeing and we'll take a look.    AFAIK, nothing has been introduced that is intentionally incompatible with IE.

**Comment by TomHuang204** [comment]
@mlrawlings ,hi, it was my mistake. I made some mistakes when using the lasso-babel-transform and the babel-polyfill. Anyway, there has no issues about the IE support. Thanks.
 By the way, I have a question, it is about the 'fingerprintsEnabled' option of the lasso. I know this is used for cache busting. Under my understanding, there will be a file with a shasum for each template after the template has been made in the first time, the browser will request this file to render the page. But under our department production env, there is a load-balance server between the server machines and the browsers of customers. So, requests from a page may not arrive at the same machine. If I set the 'fingerprintsEnabled' to true, the files(xxx.js, xxx.css) with a certain shasum will not be found. So, if there is a load-balance server between our machines and the browsers, is my only choice to set the 'fingerprintenabled' to false?
 Sorry for my poor English, thanks.
